views/layouts/main.handlebars

```diff
@@ -29,16 +29,21 @@
     <a href="https://qsurvey.mozilla.com/s3/txp-firefox-send" rel="noreferrer noopener" class="feedback" target="_blank" data-l10n-id="siteFeedback">Feedback</a>
   </header>
   <div class="all">
+    <noscript>
+      <h2>Firefox Send requires JavaScript</h2>
+      <p><a href="https://github.com/mozilla/send/blob/master/docs/faq.md#why-does-firefox-send-require-javascript" target="_blank" rel="noreferrer noopener">Why does Firefox Send require JavaScript?</a></p>
+      <p>Please enable JavaScript and try again.</p>
+    </noscript>
```

Comment:
This is all en-US only. Sorry. But since l20n doesn't work without JavaScript, we're a bit stuck unless we do weird workarounds like some meta refresh to localized copies of a page.